### PR TITLE
WEBEU2 - Responsive Web Design I - James Eneh 🇳🇬

### PR DESCRIPTION
**Pull request by erozonachi**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,16 +1,98 @@
 <!doctype html>
 
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <title>Great Idea - Responsive I</title>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (maximesalomon) [inline index.html:6]: 👍

File: index.html [hunk 2]
```diff
@@ -1,16 +1,98 @@
...
+              Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+            </p>
+          </div>
+        </section>
+      </main>
+      <hr>
+    
+      <footer>
+        <h3>Contact</h3>
+        123 Way 456 Street<br>
```
Comment (maximesalomon) [inline index.html:89]: You can use `<address>` here :)

File: services.html [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+          <li><a href="#">Vision</a></li>
+          <li><a href="#">Features</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+        <a href="index.html">
+          <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+        </a>
+      </nav>
+      <img class="services-header-img" src="img/services-header.jpg" alt="Our services header image">
```
Comment (maximesalomon) [inline services.html:34]: `<main>` here

File: css/index.css [hunk 4]
```diff
@@ -1,3 +1,375 @@
...
+    justify-content: space-around;
+  }
+  header .header-container .header-img {
+    display: none;
+  }
+  main section .lower-text-container {
+    width: 31%;
+  }
+}
+
```
Comment (maximesalomon) [inline css/index.css:283]: You might want to add more comments to your css :)

**Review by maximesalomon — COMMENTED** [review]
Fantastic job on this project James 🔥 

Try to add more comments to your css for yourself and for me :)

File: index.html (hunk 2 above)
Comment (erozonachi) [reply index.html:89]: Okay 😎

File: services.html (hunk 3 above)
Comment (erozonachi) [reply services.html:34]: Noted ✅

File: css/index.css (hunk 4 above)
Comment (erozonachi) [reply css/index.css:283]: Noted ✅

**Pull request merged**